.travis.yml

```diff
@@ -138,11 +146,12 @@ before_install:
     if [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
       export JOBS=$((`sysctl -n hw.ncpu` + 1))
       sudo mdutil -i off /
+      npm install -g yarn
```

Comment:
No it is updated automatically by yarn when you add a package using `yarn add package`. You will have to commit it though.